src/singolo.js

```diff
@@ -0,0 +1,158 @@
+// Anchors
+window.addEventListener('scroll', scroll);
```

Comment:
You shouldn't have code without additional functions
It's hard to understand why do you need/do something
Please wrap all needed functionality in functions. The main one can be called init for example.